### PR TITLE
Addressing comments about pre-commit

### PR DESCRIPTION
**Pull request by gvwilson**
Addresses #17.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -104,26 +104,29 @@ Remember that even text---often thought of as a lowest common denominator---can
...
-3.  The project repository contains a checklist of things that must pass before the merge can be done
-    *   Style guidelines met
-    *   Tests pass
-    *   To-do list updated
-4.  The repository is mirrored on at least one machine that *isn't* the researcher's laptop
+2.  The repository is mirrored on at least one machine that *isn't* the researcher's laptop
     *   E.g., pushed to GitHub or sync'd with a departmental server
+3.  The project repository contains a checklist of things that must pass before a change is shared with the world
+    *   Style guidelines met, to-do list updated, automated tests pass (if there are any)
+    *   Note: "shared with the world" means "pushed to GitHub" or however else changes are copied off the researcher's laptop
```
Comment (lexnederbragt) [inline index.md]: Laptop --> Laptop or PC

**Comment by lexnederbragt** [comment]
I like it!

**Comment by BernhardKonrad** [comment]
Good work, I think that's the best we can do here.

File: index.md (hunk 1 above)
Comment (gvwilson) [inline index.md]: Addressed in latest push.

**Pull request merged**